dtests/src/resources/scripts/northWind/nw_queriesWithChangingConstants.sql

```diff
@@ -0,0 +1,321 @@
+elapsedtime on;
```

Comment:
This should include only queries with changing constants. No need to add all other queries in the bt as it will increase test and analysis time. We are already having coverage for the queries without changing any constants in many tests.